build.gradle

```diff
@@ -549,6 +550,7 @@ task api(type: Javadoc) {
 	options.overview = 'src/api/overview.html'
 	options.stylesheetFile = file('src/api/stylesheet.css')
 	options.links(rootProject.ext.javadocLinks)
+	options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
Do you still see JavaDocs warning when you run this task?

Reply:
Yes.